### PR TITLE
login page design version 2

### PR DESCRIPTION
**Pull request by veeranjaneyuludavuluri**

### CODE REVIEW

**Review by dhaval2989 — APPROVED** [review]
good job

File: src/app/auth/login/login.component.scss [hunk 1]
```diff
@@ -40,18 +40,27 @@
 button:focus {
     outline: none !important;
 }
+
 .nxt-btn:active {
     color: #00000099;
```
Comment (Sina-pz) [inline src/app/auth/login/login.component.scss:45]: use our style

Comment (Sina-pz) [inline src/app/auth/login/login.component.scss:45]: same

File: src/app/auth/login/login.component.scss [hunk 2]
```diff
@@ -40,18 +40,27 @@
...
     background-color: #f6f7f8 !important;
     border: #5D5D5D
 }
 
+select {
+    background-image: linear-gradient(45deg, transparent 50%, #272626 60%), linear-gradient(135deg, #1f1d1d 40%, transparent 50%) !important;
```
Comment (Sina-pz) [inline src/app/auth/login/login.component.scss:51]: same

File: src/app/auth/login/login.component.scss [hunk 3]
```diff
@@ -130,8 +139,21 @@ button {
...
     margin-left: $padding-16;
+    font-family: Lato;
 }
 
 .body-divider {
     box-shadow: 0px 1px 2px;
+}
+
+.background-liner {
+    background: transparent linear-gradient(180deg, $color-white-20 0%, #D5D5D5 32%, #FFFCFC 63%, #EAEAEA 69%, #E6E6E6 100%) 0% 0% no-repeat padding-box
```
Comment (Sina-pz) [inline src/app/auth/login/login.component.scss:150]: same

**Review by Sina-pz — APPROVED** [review]
Good job

**Pull request merged**